contracts/src/StakeTableV2.sol

```diff
@@ -336,6 +346,9 @@ contract StakeTableV2 is StakeTable, PausableUpgradeable, AccessControlUpgradeab
         // Initialize undelegation IDs to start at 1 (0 is reserved for V1 undelegations)
         nextUndelegationId = 1;
 
+        // Initialize min delegate amount to 1 ESP token
+        minDelegateAmount = 1 ether;
```

Comment:
I guess theoretically we should check the decimals here but i'm also fine with a comment saying "the token is hardcoded to 18 decimals"